.github/workflows/notify-release.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           curl -XPOST \
             -H "Authorization:Bearer ${{ secrets.GH_TKN_DARWINIA }}" \
             -H "Accept:application/vnd.github+json" \
-            "https://api.github.com/repos/darwinia-network/wasm-runtime-overrides/dispatches" \
+            "https://api.github.com/repos/darwinia-network/darwinia-releases/dispatches" \
```

Comment:
It's looks wrong, we changed use workflow_dispatchs, the documents here https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#create-a-workflow-dispatch-event

May we should change to 

```
"https://api.github.com/repos/darwinia-network/darwinia-releases/actions/workflows/wasm.yaml/dispatches" \
-d '{"ref":"main","inputs":{"network":"mainnet","tag":"'${TAG}'"}}'
```